docs/runbooks/CDINotReady.md

```diff
@@ -1,32 +1,41 @@
 # CDINotReady
+<!--Edited by davozeni, 17.10.2022-->
 
 ## Meaning
 
-The CDINotReady alert is indicative of a CDI installation being in a degraded state;  
+The CDINotReady alert indicates that a containerized data importer (CDI) installation is in a degraded state:
+
 - Not progressing
+
 - Not available to use
 
 ## Impact
 
-CDI is simply not usable - users cannot build Virtual Machine Disks on PVCs using CDI's DataVolumes.  
-It's components are not ready & stopped progressing towards a ready state and hence require action.
+CDI is not usable, so users cannot build Virtual Machine Disks on PVCs using CDI's DataVolumes. 
```

Comment:
PVC should be spelled out the first time: persistent volume claim.

"Virtual Machine Disks" is not capitalized. 

I would write "DataVolumes" as "data volumes" because you are using them in a generic sense, and not as the name of a pod.

Reply:
Thanks for the feedback. I changed these terms in the second commit.